### ISSUE TITLE
Parsing error: x-invalid-end-tag  vue/no-parsing-error

### ISSUE THREAD

**Issue by Shinigami92** [rashfael/eslint-plugin-vue-pug#3]
![image](https://user-images.githubusercontent.com/7195563/164394740-2f1d231c-7cb5-4a03-aafe-627806697ae0.png)

Repor: 

```vue
<script setup lang="ts"></script>

<template lang="pug">
img(src="@/assets/logo_text.svg", height="32")
</template>
```

Since v0.4.0, not in v0.3.0

There is no fixer and no hint how to solve that
Also IMO it is valid pug code 🤔 

Workaround until fixed: `'vue/no-parsing-error': 'off'`

**Comment by rashfael** [comment]
Looks like I started generating end tags for html void elements in my rewrite.
Fixed in `vue-eslint-parser-template-tokenizer-pug@0.4.1`.

I'll also probably disable https://eslint.vuejs.org/rules/html-self-closing.html in all eslint-plugin-vue-pug configs, since that only lints tokenizer internals.

**Comment by Shinigami92** [comment]
Fixed in 0.4.1
